src/eval-frame/components/panes/__tests__/history-item.test.js

```diff
@@ -7,7 +7,6 @@ import React from "react";
 import { store } from "../../../store"; /* eslint-disable-line no-unused-vars */
 
 import HistoryItem from "../history-item";
-import PaneContentButton from "../pane-content-button";
```

Comment:
as mentioned elsewhere: we should revive these tests since we're updating this stuff